personio-personnel-data-api-oa3.yaml

```diff
@@ -2139,6 +2140,146 @@ paths:
+                                type: standard
+                                universal_id: last_name
+                              email:
+                                label: Email
+                                value: michael.miller@demo.com
+                                type: standard
+                                universal_id: email
+                          absence_type:
+                            type: AbsenceType
+                            attributes:
+                              id: "9bba303f-0fbc-4514-9958-0befa21923fb"
+                              name: "Absence Type Name"
+                          certificate:
+                            status: not-required
+                          start: "2022-05-31T22:00:00.0Z"
+                          end: "2022-06-02T22:00:00.0Z"
+                          half_day_start: true
+                          half_day_end: false
+                          comment: "this is a comment"
+                          origin: "web"
+                          status: "approved"
+                          created_by: 1
+                          created_at: "2022-06-20T14:29:56.510Z"
+                          updated_at: "2022-06-20T14:29:56.510Z"
+                          approved_at: "2022-06-20T14:29:56.510Z"
+                          breakdowns:
+                            - date: "2022-06-01"
+                              effective_duration: 480
+                            - date: "2022-06-02"
+                              effective_duration: 480
+        "400":
+          description: Invalid request
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorInsertingAbsenceResponse"
+              examples:
+                response:
+                  value:
+                    success: false
+                    error:
+                      code: 400
+                      message: 'Error when trying to insert absence period'
+
+        "404":
+          description: Employee or Absence type not found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+              examples:
+                response:
+                  value:
+                    success: false
+                    error:
+                      code: 0
+                      message: 'Absence type not found.'
+
+        "422":
+          description: Validation error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorCreateAbsenceResponse"
+              examples:
+                response:
+                  value:
+                    success: false
+                    error:
+                      code: 0
+                      message: 'The given data failed to pass validation.'
+                      error_data: '{...}'
+        "500":
```

Comment:
[Suggestion]: Feel free to ignore it, but in my opinion, we don't need to add an example for a `500` error here.

Reply:
I just left it to follow the same pattern as the GET method we have, but I got your point that it's kind of redundant 👍